### DISCUSSION TITLE
Help me find University  of Agriculture Faisalabad

### DISCUSSION THREAD

**Discussion by chyasirabbasjutt** [cu-esiil-edu/2024-data-short-course#65] [Q&A]
# Search for Unversity of Agriculture Faisalabad
uaf_gdf = osm.features_from_address(
    'Unversity of Agriculture, Faisalabad, Pakistan',
    {'amenity': ['college']})
uaf_gdf 
@eculler please help . @nquarder

**Comment by nquarder** [comment]
@chyasirabbasjutt okay try this and see if it works for you: 

`uaf_gdf = osm.features_from_address(`
`'University of Agriculture, Pakistan',`
`{'amenity': ['university']})`
